app/uk/gov/hmrc/residencenilratebandcalculator/models/GetTransitionOutPrefix.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.residencenilratebandcalculator.models
+
+import uk.gov.hmrc.residencenilratebandcalculator.Constants
+
+object GetTransitionOutPrefix {
+  def apply(userAnswers: UserAnswers) =
+    (userAnswers.dateOfDeath, userAnswers.anyEstatePassedToDescendants) match {
+      case (Some(dateOfDeath), _) if dateOfDeath isBefore Constants.eligibilityDate => "not_possible_to_use_service.date_of_death"
+      case (_, Some(false)) => "not_possible_to_use_service.direct_descendant"
+      case _ => "not_possible_to_use_service.grossing_up"
+    }
+}
```

Comment:
Yep, completely agree.  Since we're a little short on time I was hoping to do that during the refactor of the three pages?